Example/Tests/HPPManagerTests.swift

```diff
@@ -0,0 +1,86 @@
+//
+//  HPPManagerTests.swift
+//  RXPiOS_Tests
+//
+//  Copyright © 2020 realexpayments. All rights reserved.
```

Comment:
This would just be GlobalPayments. Realex Payments as an entity no longer exists.